assets/js/state/actions/lastExecutions.js

```diff
@@ -1,6 +1,12 @@
 export const UPDATE_LAST_EXECUTION = 'UPDATE_LAST_EXECUTION';
+export const EXECUTION_REQUESTED = 'EXECUTION_REQUESTED';
 
 export const updateLastExecution = (groupID) => ({
   type: UPDATE_LAST_EXECUTION,
   payload: { groupID },
 });
+
+export const executionRequested = (clusterID, hosts, checks) => ({
+  type: EXECUTION_REQUESTED,
+  payload: { clusterID, hosts, checks },
+});
```

Comment:
Maybe just `requestExecution`?

Reply:
Let's avoid this discussion now. We already have a tech debt ticket to talk about present/past term in the actions/events, etc